corehq/apps/linked_domain/static/linked_domain/js/domain_links.js

```diff
@@ -175,15 +175,19 @@ hqDefine("linked_domain/js/domain_links", [
         self.addDownstreamDomain = function (viewModel) {
             _private.RMI("create_domain_link", {
                 "downstream_domain": viewModel.value(),
-            }).done(function (data) {
-                self.availableDomains(_.filter(self.availableDomains(), function (item) {
-                    return item !== viewModel.value();
-                }));
-                self.value(null);
-                self.parent.domain_links.unshift(DomainLink(data.response));
-                self.parent.goToPage(1);
+            }).done(function (response) {
+                if (response.success) {
+                    self.availableDomains(_.filter(self.availableDomains(), function (item) {
+                        return item !== viewModel.value();
+                    }));
+                    self.value(null);
+                    self.parent.domain_links.unshift(DomainLink(response.domain_link));
+                    self.parent.goToPage(1);
+                } else {
+                    alertUser.alert_user(gettext('Unable to link project spaces. ' + response.message + '\nYou must remove the existing link before creating this new link.'), 'danger');
```

Comment:
This won't translate because it's dynamic. We often use `_.template` in js similarly to python's `format` to build translate-able strings.